src/Twig/Handler/RecordHandler.php

```diff
@@ -313,4 +314,26 @@ public function selectField($content, $fieldName, $startempty = false, $keyName
 
         return $retval;
     }
+
+    /**
+     * Return whether or not a given path resolves, i.e. if it has a defined route.
+     *
+     * @param string $name
+     * @param array  $parameters
+     * @param bool   $relative
+     *
+     * @return bool
+     */
+    public function isPath($name, $parameters = array(), $relative = false)
+    {
+        $success = true;
+
+        try {
+            $path = Library::path($name, $parameters, $relative);
+        } catch (\Exception $e) {
```

Comment:
Also, just came back and noticed this, we should catch `\Symfony\Component\Routing\Exception\RouteNotFoundException` as a general exception could point to other problems in the stack that would be silently buried.